src/components/SavedNewsList/SavedNewsList.js

```diff
@@ -0,0 +1,41 @@
+import { useContext, useEffect } from "react";
+
+import "../SavedNews/SavedNews.css";
+import NewsCard from "../NewsCard/NewsCard";
+import { CurrentUserContext } from "../../contexts/CurrentUserContext";
+import { SavedNewsContext } from "../../contexts/SavedNewsContext";
+import { getSavedNews } from "../../utils/api";
+
+const SavedNewsList = ({
+  handleSaveCard,
+  handleDeleteCard,
+  onLogin,
+  isSaved,
+}) => {
+  const { currentUser } = useContext(CurrentUserContext);
+  const { savedNews, setSavedNews } = useContext(SavedNewsContext);
+
+  useEffect(() => {
+    const jwt = localStorage.getItem("token");
+    getSavedNews(jwt).then(setSavedNews);
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**